src/main/java/Duke.java

```diff
@@ -1,10 +1,191 @@
+import java.util.Scanner;
+import java.util.Arrays;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    private static void writeToFile(String filePath, String[] lists, String[] taskType, String[] doneTasks, String[] dates, int number) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(int i = 0; i < number; i++) {
+            fw.write(taskType[i] + " | " + doneTasks[i] + " | " + lists[i] + " | " + dates[i] + "\n" );
+        }
+        fw.close();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final String DIVIDER = "    ——————————————————————————————————————————————————————————————";
+        System.out.println(DIVIDER);
+        System.out.println("    Hello! I'm Duke\n    What can I do for you?");
+        System.out.println(DIVIDER);
+        String line;
+        String[] lists = new String[100];
+        String[] taskType = new String[100];
+        String[] doneTasks = new String[100];
```

Comment:
Consider creating a TaskManager class to store all tasks list and related manipulation methods